model/src/pyrenew/process/randomwalk.py

```diff
@@ -101,3 +99,39 @@ def validate():
         """
         super().validate()
         return None
+
+
+class StandardNormal(RandomWalk):
```

Comment:
Suggest including RandomWalk in the name.

Reply:
Yes, this was a mistake. Fixed in 8a4f6c6